src/util/heap/layout/mmapper.rs

```diff
@@ -157,6 +168,7 @@ impl MapState {
         state_slices: &[&[Atomic<MapState>]],
         mmap_start: Address,
         strategy: MmapStrategy,
+        anno: &MmapAnnotation,
```

Comment:
Need to update the doc comment to mention the new argument

Reply:
Done.